### PR TITLE
Formatting

### PR DESCRIPTION
**Pull request by pajamian**

### CODE REVIEW

**Comment by pajamian** [comment]
@SergeCroise Did I miss anything?

File: migrate2rocky/migrate2rocky.sh [hunk 1]
```diff
@@ -144,23 +145,23 @@ exit_message() {
 
 final_message() {
     errmsg '%s ' \
-	"An error occurred while we were attempting to convert your system to" \
-	"Rocky Linux. Your system may be unstable. Script will now exit to" \
-	"prevent possible damage."$'\n\n'
+        "An error occurred while we were attempting to convert your system to" \
+        "Rocky Linux. Your system may be unstable. Script will now exit to" \
```
Comment (SergeCroise) [inline migrate2rocky/migrate2rocky.sh:149]: missing blank at the beginning of the line?
\" Rocky Linux. Your system may be unstable. Script will now exit to...
(or at the end of the previous)

File: migrate2rocky/migrate2rocky.sh [hunk 2]
```diff
@@ -144,23 +145,23 @@ exit_message() {
...
+        "prevent possible damage."$'\n\n'
```
Comment (SergeCroise) [inline migrate2rocky/migrate2rocky.sh:150]: \" prevent possible damage...

File: migrate2rocky/migrate2rocky.sh [hunk 3]
```diff
@@ -173,84 +174,92 @@ os-release () (
...
+'Migration from Uyuni/SUSE Manager-modified systems is not supported by '\
+'migrate2rocky. See the README file for details.'
     fi
 }
 
 # All of the binaries used by this script are available in a EL8 minimal install
 # and are in /bin, so we should not encounter a system where the script doesn't
-# work unless it's severly broken.  This is just a simple check that will cause
+# work unless it's severely broken.  This is just a simple check that will cause
 # the script to bail if any expected system utilities are missing.
```
Comment (SergeCroise) [inline migrate2rocky/migrate2rocky.sh:220]: ... to bail ?
see also:
function:
https://github.com/rocky-linux/rocky-tools/blob/c53d1488fdedb84928eee29d980be96811d8232f/migrate2rocky/migrate2rocky.sh#L358
and 
highlighted in:
https://github.com/rocky-linux/rocky-tools/blob/c53d1488fdedb84928eee29d980be96811d8232f/migrate2rocky/README.md
...is run.  The RockyLinux kernel should come up as the default highlighed kernel...

File: migrate2rocky/migrate2rocky.sh [hunk 4]
```diff
@@ -477,178 +491,188 @@ collect_system_info () {
...
 "In addition to the above the following system packages will be removed:" \
-	    "${addl_pkg_removes[@]}"
+            "${addl_pkg_removes[@]}"
     fi
 
     # Release packages that are part of SIG's should be listed below when they
     # are available.
     # UPDATE: We may or may not do something with SIG's here, it could just be
-    # left as a separate excersize to swap out the sig repos.
+    # left as a separate excercise to swap out the sig repos.
```
Comment (SergeCroise) [inline migrate2rocky/migrate2rocky.sh:620]: ... exercise ...

**Review by SergeCroise — APPROVED** [review]

File: migrate2rocky/migrate2rocky.sh (hunk 1 above)
Comment (pajamian) [reply migrate2rocky/migrate2rocky.sh:149]: Blank is in the format specifier '%s '

File: migrate2rocky/migrate2rocky.sh (hunk 2 above)
Comment (pajamian) [reply migrate2rocky/migrate2rocky.sh:150]: ...and again.

File: migrate2rocky/migrate2rocky.sh (hunk 3 above)
Comment (pajamian) [reply migrate2rocky/migrate2rocky.sh:220]: Don't see bail, got function and highlighted.

File: migrate2rocky/migrate2rocky.sh (hunk 4 above)
Comment (pajamian) [reply migrate2rocky/migrate2rocky.sh:620]: oh geez ... got it.

File: migrate2rocky/migrate2rocky.sh (hunk 3 above)
Comment (SergeCroise) [reply migrate2rocky/migrate2rocky.sh:220]: # the script to bail if any expected system utilities are missing.

Comment (SergeCroise) [reply migrate2rocky/migrate2rocky.sh:220]: https://github.com/rocky-linux/rocky-tools/blob/7f08a2b69635223e5d8532b30c4580f528558571/migrate2rocky.sh#L62

**Pull request merged**